packages/site/root/styles.css

```diff
@@ -0,0 +1,103 @@
+body{
```

Comment:
Can you do a quick formatting pass on this file? The things that jump out at me are:
1. inconsistent spacing before `{` blocks 
2. inconsistent spacing after `:`s
3. inconsistent indentation

Longer term we should enable linting or prettier on CSS in this directory but that needn't block this submission.

Reply:
cleaned up in https://github.com/ni/nimble/pull/800/commits/0ecc3883eeebfcc1ca6b9615d2fabca347f21a66